RealmSwift/Sync.swift

```diff
@@ -1039,4 +1021,19 @@ public extension MongoCollection {
         }
     }
 }
+
+// MARK: Dictionary
+fileprivate extension Dictionary {
```

Comment:
I'm still not sure this is necessary.

Reply:
You could add a nested function to do this logic, might be clearer as this extension provides very little benefit to the codebase?